src/infra/accounts/mysql/tests/repos/test_mysql_account_repository.rs

```diff
@@ -7,67 +7,71 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-use database_common::{run_transactional, MySqlTransactionManager};
+use database_common::{DatabaseTransactionRunner, MySqlTransactionManager};
 use dill::{Catalog, CatalogBuilder};
 use kamu_accounts_mysql::MySqlAccountRepository;
 use sqlx::MySqlPool;
 
-/////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////
 
 #[test_group::group(database, mysql)]
 #[test_log::test(sqlx::test(migrations = "../../../../migrations/mysql"))]
 async fn test_missing_account_not_found(mysql_pool: MySqlPool) {
     let harness = MySqlAccountRepositoryHarness::new(mysql_pool);
 
-    run_transactional(&harness.catalog, |catalog: Catalog| async move {
+    <DatabaseTransactionRunner>::run_transactional(&harness.catalog, |catalog| async move {
```

Comment:
Why in some places we call it WITH and WITHOUT brackets? Are you sure these brackets are not an artifact of your IDE, as I can't see why they would be required?

Reply:
> Are you sure these brackets are not an artifact of your IDE, as I can't see why they would be required?

yes, I'm sure

> Why in some places we call it WITH and WITHOUT brackets?

This is based on the default error type. I used a similar approach once, while we were rethinking the Visitors API

```rust
pub struct DatabaseTransactionRunner<HFutResultE = InternalError> {
    _phantom: PhantomData<HFutResultE>,
}
```